lib/api/actions/functions.php

```diff
@@ -5,9 +5,13 @@
  * While WordPress requires two or three arguments to remove an action, Beans
  * actions can be modified, replaced, removed or reset using only the ID as a reference.
  *
- * @package API\Actions
+ * @package Beans\Framework\API\Actions
+ *
+ * @since   1.5.0
  */
 
+// @codingStandardsIgnoreStart - Skipping over non-compliant code.
```

Comment:
This should not be added, if code changes, they need to be compliant.

Reply:
@ThierryA We have to skip over the functions that have not yet been converted and made compliant.  Else, we have to convert the entire file and all of the functions for each and every PR.  The idea is to break up PRs into smaller chunks in order to ease the code review process.

This PR is just for the 2 functions only and _all of the API actions_ functions.

Reply:
The code sniffer only picks up changes, so you shouldn't have to add that if the functions aren't changed.

Reply:
@ThierryA Actually it failed the PHPCS tests, which is why I put those 2 lines in.

Reply:
All fixed now, the dev-lib was incorrectly setup to check changed files instead of only patches.